keps/sig-api-machinery/4008-crd-ratcheting/README.md

```diff
@@ -868,8 +858,10 @@ which tightens a validation.
 - CEL expressivity to specify custom ratcheting validation
 
 #### GA
-- Upgrade/Downgrade e2e tests
-- Scalability Tests
+- [x] Upgrade/Downgrade e2e tests
+- [ ] Scalability Tests
```

Comment:
This was listed here from the original KEP. Hoping to see agreement about what scalability tests are required for this. Appreciate guidance from reviewers. Ratcheting adds an extra `DeepEqual` comparison to validation - and it is important the implementation remembers the result of the comparison for each subfield rather than duplicating work.